app/domain/object/timeline.go

```diff
@@ -0,0 +1,6 @@
+package object
+
+type Timeline struct {
+	Account *Account
+	Status  *Status
+}
```

Comment:
さて、[このレビュー](https://github.com/sekibouya/yatter-backend-go/pull/3#discussion_r1751672830) で `Account` の情報はすでに `Status` 構造体に組み込まれているため削除できます。

また、実際のタイムラインは複数の `status` が表示されることが想定されるので、スライスにしておきましょう。

```suggestion
type Timeline struct {
	Status  []*Status
}
```